src/main/java/orca/flukes/ndl/ManifestLoader.java

```diff
@@ -388,85 +387,17 @@ public void ndlLinkConnection(Resource l, OntModel m,
 			setRequestGuid(l, ml);
 			
 			nodes.put(getTrueName(l), ml);
-			
-			// special case handling - if storage is one side of the link
-			// (only for shared vlan storage), then we only remember
-			// interfaces with no ip addresses on them (others are duplicates; 
-			// we save the equivalence relationship to lookup IP later) /ib 09/18/14
-			boolean sharedVlanStorageLink = false;
-			Map<Resource, List<Resource>> interfacesByOwner = new HashMap<Resource, List<Resource>>();
-			
-			for(Resource ti: interfaces) {
-				List<Resource> attached = NdlCommons.getWhoHasInterface(ti, m);
-				if (attached == null)
-					continue;
-				for(Resource ta: attached) {
-					// skip links (self included)
-					if (NdlCommons.hasResourceType(ta, NdlCommons.topologyLinkConnectionClass))
-						continue;
-					if (NdlCommons.isISCSINetworkStorage(ta)) {
-						sharedVlanStorageLink = true;
-					}
-					List<Resource> tmp = interfacesByOwner.get(ta);
-					if (tmp == null)
-						tmp = new ArrayList<Resource>();
-					tmp.add(ti);
-					interfacesByOwner.put(ta, tmp);
-				}
-			}
-
-			// map ip/noip interfaces to each other
-			for(Map.Entry<Resource, List<Resource>> ee: interfacesByOwner.entrySet()) {
-				// for regular nodes and storage, not for node groups
-				if (ee.getValue().size() == 2) {
-					Resource noIp = null, ip = null;
-					if (NdlCommons.getInterfaceIP(ee.getValue().get(0)) == null) {
-						noIp = ee.getValue().get(0);
-						ip = ee.getValue().get(1);
-					} else {
-						noIp = ee.getValue().get(1);
-						ip = ee.getValue().get(0);
-					}
-					sharedStorageLinkInterfaceEquivalence.put(noIp, ip);
-					sharedStorageLinkInterfaceEquivalence.put(ip, noIp);
-				}
-			}
 
 			// remember the interfaces
 			while(it.hasNext()) {
 				Resource intR = it.next();
-				//interfaceToNode.put(getTrueName(intR), ml);
-				if (sharedVlanStorageLink) {
-					// does it have an IP address? - then we use it (for node groups this ends up false)
-					if ((NdlCommons.getInterfaceIP(intR) != null) && sharedStorageLinkInterfaceEquivalence.containsKey(intR)) {
```

Comment:
1. It starts here - this IF no longer works properly and the skipping of the interface as the log message says isn't needed.